src/vs/vscode.d.ts

```diff
@@ -2482,6 +2482,11 @@ declare module 'vscode' {
 		appendVariable(name: string, defaultValue: string | ((snippet: SnippetString) => any)): SnippetString;
 	}
 
+	export interface RenameInitialValue {
```

Comment:
Needs maybe some better naming here but idea is good

Reply:
I am actually wondering when we would return both, so range and text (which then would be different from the text in range?). Do you have a use-case in mind?